src/Engine/ProtoCore/ProcedureTable.cs

```diff
@@ -299,6 +299,19 @@ public IEnumerable<ProcedureNode> GetFunctionsByName(string name)
             return Procedures.Where(p => p.Name.Equals(name));
         }
 
+        /// <summary>
+        /// Returns getter function for given property name.
+        /// Ex: for X property of Point, this method returns get_X(Point p).
+        /// </summary>
+        /// <param name="name"></param>
+        /// <returns></returns>
+        public ProcedureNode GetPropertyGetterByName(string name)
+        {
+            var getterMethodName = Constants.kGetterPrefix + name;
+            return Procedures.FirstOrDefault(p => p.Name == getterMethodName &&
```

Comment:
does this potentially search the entire procedure table?
Is this class actually not a table????

Reply:
This searches only the procedure table for the particular class.

Reply:
funny we call it a table...

Reply:
That is the typical terminology you will find in compilers.